core/data/build.gradle.kts

```diff
@@ -40,6 +51,15 @@ dependencies {
 	implementation(libs.androidx.appcompat)
 	implementation(libs.kotlinx.serialization.json)
 	implementation(libs.material)
+	implementation(libs.timber)
+
+	// Ktor
+	implementation(libs.ktor.client.auth)
+	implementation(libs.ktor.client.cio)
+	implementation(libs.ktor.client.content.negotiation)
+	implementation(libs.ktor.client.core)
+	implementation(libs.ktor.client.logging)
+	implementation(libs.ktor.serialization.kotlinx.json)
```

Comment:
You could make those dependencies a bundle in your version catalog, so you can apply all these with a single line in each module that needs Ktor